addon/templates/components/frost-list-content-container.hbs

```diff
@@ -1,44 +1,31 @@
 {{! Template for the frost-list-content-container component }}
+{{#vertical-collection
+  alwaysRemeasure=true
+  alwaysUseDefaultHeight=alwaysUseDefaultHeight
+  bufferSize=bufferSize
+  content=items
+  defaultHeight=defaultHeight
+  firstReached=onLoadPrevious
+  lastReached=onLoadNext
+  scrollPosition=scrollTop
+  useContentProxy=false
 
-{{#if pagination}}
```

Comment:
Is it necessary to remove the "basic" {{#each}} loop in the pagination case?  I'd prefer to not introduce {{#vertical-collection}} for those people using the pagination scenario in a minor release